desktop_server/src/com/mygdx/game/server/initialize/StartUnitInitializer.java

```diff
@@ -32,11 +36,21 @@ public void initializeStartingUnits(IntArray map) {
     for (int i = 0; i < gameRoom.getNumberOfClients(); i++) {
       var client = gameRoom.getClients().get(i);
       log.info("client id " + client.getPlayerToken());
+      var config = getCivConfig(client.getCivId());
       var fieldId = getUnusedField(map);
-      createUnitService.createUnit(anyConfig, fieldId, client, true);
+      createUnitService.createUnit((int) config, fieldId, client, true);
```

Comment:
czyli nie ma tutaj testu na to, żeby klient/gracz tworzył jednostki swojej cywilizacji? ale w sumie to teraaz nie musi takiego czegoś być

Reply:
popatrz wyżej, config biorę odpowiedni dla danej cywilizacji jaką ma gracz